### ISSUE TITLE
Add a status bar to show topology statistics

### ISSUE THREAD

**Issue by tomwilkie** [weaveworks/scope#207]
When we start showing containers without connections, we'll want an option to not do that.

Also means we could start probing for processes without connections, and optionally show/hide them.

**Comment by tomwilkie** [comment]
Also lets not forget to show the number of unconnected nodes that have been filtered out.

**Comment by davkal** [comment]
Maybe just as one string:
```
[12 nodes, 15 connections, 4 unconnected nodes (show)]
```
with `show` being a link.

**Comment by davkal** [comment]
Reopened, to track count of filtered nodes in status.

@tomwilkie need backend support for this in the stats object:
```
"stats":{"node_count":12,"nonpseudo_node_count":11,"edge_count":11}}
```

Or what is `nonpseudo_node_count`, meaning what would be a human readable label for that?

**Comment by tomwilkie** [comment]
@davkal this is done now?  Anything left?

**Comment by davkal** [comment]
@tomwilkie Nope, all good. Fixed via #519
